plugins/hotCards/hotCards.js

```diff
@@ -159,7 +158,7 @@
    */
   function handleScenesHotCards() {
     const pattern =
-      /^\/(scenes$|performers\/\d+\/scenes$|studios\/\d+\/scenes$|tags\/\d+\/scenes$|movies\/\d+$|galleries\/\d+$)/;
+      /^\/(scenes|(performers|studios|tags|movies)\/\d+\/scenes|(movies|galleries)\/\d+)$)/;
```

Comment:
I think you left an extra ')' right at the end. I just updated the plugin and I'm getting "SyntaxError: unmatched ) in regular expression". Thank you.

Reply:
#350